operator/gefyra/connection/stowaway/resources/configmaps.py

```diff
@@ -19,6 +20,7 @@ def create_stowaway_proxyroute_configmap() -> k8s.client.V1ConfigMap:
 
 
 def create_stowaway_configmap() -> k8s.client.V1ConfigMap:
+    stowaway_ip = requests.get("https://icanhazip.com").text.strip()
```

Comment:
Sorry, that does not make any sense.

Reply:
Meaning you want the requirement of icanhazip completely removed? What would the alternative be?